src/robot/romi-robot.ts

```diff
@@ -122,14 +164,22 @@ export default class WPILibWSRomiRobot extends WPILibWSRobotBase {
             if (romiConfig.gyroFilterWindowSize !== undefined) {
```

Comment:
To make the coding style more consistent and to be more accurate, should we also use if(romiConfig.gyroFilterWindowSize)? I'm not sure if 0 would ever make sense (or a negative number for that matter). Maybe if(romiConfig.gyroFilterWindowSize && romiConfig.gyroFilterWindowSize > 0)

Reply:
Since this wasn't part of this PR change, I'll resolve. Just a thought to make the code a little cleaner along the way.